app/src/main/java/org/jellyfin/androidtv/playback/PlaybackController.java

```diff
@@ -106,9 +106,8 @@ public PlaybackController(List<BaseItemDto> items, IPlaybackOverlayFragment frag
         useVlc = mApplication.getUserPreferences().getVideoPlayer() == PreferredVideoPlayer.VLC;
     }
 
-    public void init(VideoManager mgr, View spinner) {
+    public void init(VideoManager mgr) {
```

Comment:
So should it be restored here also?

Reply:
Did some digging and I removed it in commit ec28d32eb13b2a8cdbfcffe318ae1299739ef266 with this message:

> **Remove custom_player_interface.xml**
> 
> It was unused and caused confusion about which xml was used for the player (spoiler: it is the VLC one, even for exoplayer). For some reason a view was still used (spinner) so I did the most basic removal of it

So what happend is that I removed the unused xml file but because of _legacy_ the init function required a reference to the spinner that didn't exists (lol).

The `findViewById` function would always return null.

Reply:
So it can stay like this